packages/plugin-initialize-camera/src/index.ts

```diff
@@ -0,0 +1,152 @@
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
+
+const info = <const>{
+  name: "initialize-camera",
+  parameters: {
+    /** Message to display with the selection box */
+    device_select_message: {
+      type: ParameterType.HTML_STRING,
+      default: `<p>Please select the camera you would like to use.</p>`,
+    },
+    /** Label to use for the button that confirms selection */
+    button_label: {
+      type: ParameterType.STRING,
+      default: "Use this camera",
+    },
+    /** Set to `true` to include audio in the recording */
+    include_audio: {
```

Comment:
Yeah, I did think about whether it made sense to switch to one A/V plugin and I think that is probably good to do eventually. I think right now if you only have one microphone then it might fail on the second request. I haven't tested it yet.

Reply:
I'm gonna test this now.

Reply:
Yeah, setting `include_audio: true` and then also starting a microphone seems to result in the video data not existing at all. The audio recording still works. I think it is OK to release like this for now, and maybe an integrated A/V initialization can fix this in the future.

Reply:
Thanks for testing, I agree this is something that could be fixed in the future, either by combining the plugins or maybe just catching this with a console warning.